mgo/find_test.go

```diff
@@ -28,6 +28,7 @@ func TestFind(t *testing.T) {
 		executeCursorQuery func(db *mgo.Database, collectionName string, query []bson.M, sort []string, limit int, collation *mgo.Collation, results interface{}) error
 		expectedCursor     Cursor
 		expectedErr        error
+		pointer            bool
```

Comment:
You are not using this variable. Could you cover the value and pointer cases?

Reply:
Oh, yeah I solved it with reflection instead! Most of the unit-tests use value and one of the unit-tests use pointer.